AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/fields/BasicImageFieldController.kt

```diff
@@ -471,6 +477,28 @@ class BasicImageFieldController : FieldControllerBase(), IFieldController {
         Timber.i("handleSelectImageIntent() Decoded image: '%s'", imagePath)
     }
 
+    private fun directImageIntent(imageUri: Uri?) {
+        if (imageUri != null) {
+            val internalizedPick = internalizeUri(imageUri)
+            if (internalizedPick == null) {
+                Timber.w("handleSelectImageIntent() unable to internalize image from Uri %s", imageUri)
+                showSomethingWentWrong()
+                return
+            }
+
+            val imagePath = internalizedPick.absolutePath
+            viewModel = ImageViewModel(imagePath, getUriForFile(internalizedPick))
+            setTemporaryMedia(imagePath)
+            imageFileSizeWarning.visibility = View.GONE
+
+            setPreviewImage(viewModel.imagePath, maxImageSize)
+
+            Timber.i("handleSelectImageIntent() Decoded image: '%s'", imagePath)
```

Comment:
Aim to not log paths in `Timber.i`, as these can be sent to ACRA

```suggestion
            Timber.d("handleSelectImageIntent() Decoded image: '%s'", imagePath)
```